pkgs/applications/editors/emacs/elisp-packages/manual-packages/evil-markdown/default.nix

```
@@ -1,34 +1,42 @@
 { lib
-, trivialBuild
-, fetchFromGitHub
-, emacs
 , evil
+, fetchFromGitHub
 , markdown-mode
+, melpaBuild
+, writeText
 }:
 
-trivialBuild rec {
+let
   pname = "evil-markdown";
-  version = "0.pre+unstable=2021-07-21";
-
+  ename = "evil-markdown";
+  version = "20210721.723";
   src = fetchFromGitHub {
     owner = "Somelauw";
     repo = "evil-markdown";
     rev = "8e6cc68af83914b2fa9fd3a3b8472573dbcef477";
     hash = "sha256-HBBuZ1VWIn6kwK5CtGIvHM1+9eiNiKPH0GUsyvpUVN8=";
   };
+in
+melpaBuild rec {
```

I guess one purpose of your `let ... in` is to avoid `rec` and this one is indeed redundant.

```suggestion
melpaBuild {
```